### ISSUE TITLE
Make seamless updates possible

### ISSUE THREAD

**Issue by jeff-zucker** [solid-contrib/pivot#25]
I currently have Data-Kitchen (the stand-alone SolidOS running simultaneously on local and remote file systems) running on NSS.  I previously had it on CSS but switched because there was no way I could count on updates.  It uses a very specific config script which needs to run on installation but the config for CSS changes its @context settings with each version so I have no option but to manually install CSS and distribute that to users of Data-Kitchen.  I would love to go back to CSS/Pivot but I hope the updating path becomes easier.

**Comment by jeff-zucker** [comment]
Ideally, I'd like to simply include pivot as a dependency in Data-Kitchen's package.json and then have the start script fire up electron, then pivot using a config file in Data-Kitchen's install folder.

**Comment by timbl** [comment]
Yes, please make the configuration language a stable slowly changing thing, so in most cases the same config file can be used by many different versions.   Even maybe share it with other servers like NSS

**Comment by michielbdejong** [comment]
Makes sense!
It depends on how much you want to configure. The config needed for solidcommunity.net is [part of Pivot](https://github.com/solid-contrib/pivot/blob/main/config/prod.json) so then the only things you need to configure are:
* data folder
* httpsKey  / httpsCert
* port
* public-facing domain name

Pro: You can pass these 4 things directly on the command line and don't need to worry about config files, dependency versions, or contexts.
Con: This means you get subdomains, https, and mashlib, which may not be what you want on localhost.


If you tell me what your CSS config for Data-Kitchen was then we can include it in Pivot and version it here for you, would that work?
